src/deployment/flavors.md

```diff
@@ -3,41 +3,235 @@ title: Creating flavors for Flutter
+
+## Environment set up
+Prerequisites:
+* Xcode installed
+* An existing Flutter project 
+
+To set up flavors in iOS, you’ll define build configurations in Xcode. 
+
+## Creating flavors in iOS
+
+<ol markdown="1">
+<li markdown="1">
+
+Open your project in Xcode.
+
+</li>
+<li markdown=1>
+
+Select **Runner** > **New Scheme** from the menu to add a new `Scheme`.  
+* A scheme describes how Xcode runs different actions. 
+  For the purposes of this guide, the example _flavor_ and _scheme_ are 
+  named `free`. 
+  The build configurations in the `free` scheme 
+  have the `-free` suffix. 
+
+</li>
+<li markdown="1">
+
+Duplicate the build configurations to differentiate between the 
+default configurations that are already available and the new configurations 
+for the `free` scheme. 
+* Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name (Debug, Release, and Profile). 
+Duplicate the existing configurations, once for each environment. 
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+{{site.alert.note}}
+  Your configurations should be based on your **Debug.xconfig** or **Release.xcconfig**
+  file, not the **Pods-Runner.xcconfigs**. You can check this by expanding the configuration names. 
+{{site.alert.end}}
+
+</li>
+<li markdown="1">
+
+To match the free flavor, add `-free` 
+at the end of each new configuration name. 
+
+</li>
+<li markdown="1">
+
+Change the `free` scheme to match the build configurations already created.
+* In the **Runner** project, click **Manage Schemes…** and a pop up window opens. 
+* Double click the free scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its free build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step-5-ios-scheme-free.png){:width="100%"}
+
+</li>
+</ol>
+
+## Using flavors in iOS
+
+Now that you’ve set up your free flavor, 
+you can, for example, add different product bundle identifiers per flavor. 
+A _bundle identifier_ uniquely identifies your application. 
+In this example, we set the **Debug-free** value to equal 
+`com.flavor-test.free`. 
+
+<ol markdown="1">
+<li markdown="1">
+
+Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.free` to each -free scheme value.
+
+![Step 1 using flavors image.](/assets/images/docs/flavors/step-1-using-flavors-free.png){:width="100%"}  
+
+</li>
+<li markdown=1>
+
+In the **Build Settings**, set the **Product Name** value to match each flavor. 
+For example, add Debug Free. 
+
+![Step 2 using flavors image.](/assets/images/docs/flavors/step-2-using-flavors-free.png){:width="100%"}  
+
+</li>
+<li markdown=1>
+
+Add the display name to **info.plist**. Create a new key in **info.plist** called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
```

Comment:
There should already be a `Bundle Display Name` key in that file (casing is `Info.plist`).  How about:
```suggestion
Add the display name to **Info.plist**. Update the **Bundle Display Name** 
value to `$(PRODUCT_NAME)`. 
```

Reply:
Thanks Jenn! I've made the changes and will address the screenshots with #7979